src/backend/web/routes/login.js

```diff
@@ -13,7 +13,7 @@ router.use(cookieParser());
 router.use(bodyParser.urlencoded({ extended: false }));
 router.use(bodyParser.json());
 
-const { SAML2_CLIENT_SECRET } = process.env;
+const SAML2_CLIENT_SECRET = process.env.SAML2_CLIENT_SECRET || 'secret';
```

Comment:
This seems like a security issue.  Let's not provide any default fallback if we're missing a secret.  We should fail loudly if we are.

Reply:
For sure... done: https://github.com/Seneca-CDOT/telescope/pull/550/commits/e4c5a12a32b21a76efdeaf8feede880ddb3f31d6

Currently, `env.example` mentions `secret` to be the default value of `SAML2_CLIENT_SECRET`:
https://github.com/Seneca-CDOT/telescope/blob/08c55d4d694cf7b8a2c37b7945606c792976191d/env.example#L34

Let me know if you'd like me to remove this piece of information, as well.

I'll also ask if you'd like me to do the same for `SAML2_CLIENT_ID`—its default value is likewise mentioned:
https://github.com/Seneca-CDOT/telescope/blob/08c55d4d694cf7b8a2c37b7945606c792976191d/env.example#L31